src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -88,9 +94,10 @@ def handle_exception(ex):  # pylint: disable=too-many-return-statements
         if isinstance(ex, ValidationError):
             logger.error('validation error: %s', ex)
             return 1
-        if isinstance(ex, ClientRequestError):
+        if isinstance(ex, (ClientRequestError, SSLError)):
```

Comment:
Capture `requests.exceptions.SSLError` which is generated by directly calling `requests`'s methods.

Reply:
Will msrest capture SSLError?

Reply:
Yes. It will be thrown as `ClientRequestError`.

![image](https://user-images.githubusercontent.com/4003950/89989823-0f787d00-dcb4-11ea-9be0-1c7c297af44d.png)